CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## TBD - TBD
+- Add optional auditLevel parameter to IQueryRequestOptions 
```

Comment:
nit: looks like you called it `auditBehavior` in the interface.

Reply:
Fixed. Thanks.